packages/apps/fortune/exchange-oracle/server/src/modules/job/job.service.spec.ts

```diff
@@ -325,6 +327,25 @@ describe('JobService', () => {
       expect(assignment.status).toBe(AssignmentStatus.VALIDATION);
     });
 
+    it('should fail if assignment status is not ACTIVE', async () => {
+      const assignment = {
+        jobId: 1,
+        workerAddress: workerAddress,
+        status: AssignmentStatus.CANCELED,
+      };
```

Comment:
```suggestion
      assignment.status = AssignmentStatus.CANCELED;
```
There is an assignment already defined at the beginning of the tests. You only have to modify its status